compiler/luci/pass/src/PropagateConcatenationQparam.test.cpp

```diff
@@ -36,6 +37,14 @@ void addQuantParam(luci::CircleNode &node, const std::vector<float> &scale,
   node.quantparam(std::move(quantparam));
 }
 
+int quantize(float f, luci::CircleQuantParam *qparam)
```

Comment:
return type is `int`. in body `zp` is `int64_t`. Maybe use same type `int64_t`(or `int32_t`) for consistancy?

Reply:
I've changed it to `int32_t` for consistency. Thanks.